x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -28,6 +30,15 @@ import { Provider, alertsPageStateContainer, useAlertsPageStateContainer } from
 import './styles.scss';
 import { WorkflowStatusFilter } from './workflow_status_filter';
 import { AlertsDisclaimer } from './alerts_disclaimer';
+import { loadAlertAggregations } from '../../../../../plugins/triggers_actions_ui/public';
+
+interface AlertStatsState {
```

Comment:
@claudiopro These are rules stats and not alerts stats instead. I know there are many parts of the code where these terms are used wrongly. Let's refactor this to `RulesStatsState` instead

Reply:
Yes there's a little confusion starting from the imports 😅 I'll address